wdl-lint/RULES.md

```diff
@@ -8,6 +8,7 @@ be out of sync with released packages.
 
 | Name                             | Tags                          | Description                                                                           |
 | :------------------------------- | :---------------------------- | :------------------------------------------------------------------------------------ |
+| `BlanksBetweenElements`          | Spacing                       | Ensures proper blank space between elements                                           |
```

Comment:
Maybe a nitpick, but I would prefer `BlankLinesBetweenElements`. I'm not sure what the discussion said, so it can move forward if others feel otherwise.

Reply:
fine with me 👍